compliance_checker/glider_dac.py

```diff
@@ -0,0 +1,723 @@
+            'profile_time' : {
+                'dtype' : '<f8',
+                'fields' : [
+                    '_FillValue',
+                    'comment',
+                    'long_name',
+                    'observation_type',
+                    'platform',
+                    'standard_name',
+                    'units'
+                ]
+            },
+            'profile_lat' : {
+                'dtype' : '<f8',
+                'fields' : [
+                    '_FillValue',
+                    'comment',
+                    'long_name',
+                    'observation_type',
+                    'platform',
+                    'standard_name',
+                    'units',
+                    'valid_min',
+                    'valid_max'
+                ]
+            },
+            'profile_lon' : {
+                'dtype' : '<f8',
+                'fields' : [
+                    '_FillValue',
+                    'comment',
+                    'long_name',
+                    'observation_type',
+                    'platform',
+                    'standard_name',
+                    'units',
+                    'valid_min',
+                    'valid_max'
+                ]
+            },
+            'lat_uv' : {
+                'dtype' : '<f8',
+                'fields' : [
+                    '_FillValue',
+                    'comment',
+                    'long_name',
+                    'observation_type',
+                    'platform',
+                    'standard_name',
+                    'units',
+                    'valid_min',
+                    'valid_max'
+                ]
+            },
+            'lon_uv' : {
+                'dtype' : '<f8',
+                'fields' : [
+                    '_FillValue',
+                    'comment',
+                    'long_name',
+                    'observation_type',
+                    'platform',
+                    'standard_name',
+                    'units',
+                    'valid_min',
+                    'valid_max'
+                ]
+            },
+            'u' : {
+                'dtype' : '<f8',
+                'fields' : [
+                    '_FillValue',
+                    'comment',
+                    'long_name',
+                    'observation_type',
+                    'platform',
+                    'standard_name',
+                    'units',
+                    'valid_min',
+                    'valid_max'
+                ]
+            },
+            'v' : {
+                'dtype' : '<f8',
+                'fields' : [
+                    '_FillValue',
+                    'comment',
+                    'long_name',
+                    'observation_type',
+                    'platform',
+                    'standard_name',
+                    'units',
+                    'valid_min',
+                    'valid_max'
+                ]
+            }
+        }
+
```

Comment:
@daf I really like this pattern, think I'm gonna use it more often for validation stuff.